exchanges/binanceus/binanceus_wrapper.go

```diff
@@ -0,0 +1,976 @@
+	return resp, nil
+}
+
+// GetHistoricTrades returns historic trade data within the timeframe provided
+func (bi *Binanceus) GetHistoricTrades(ctx context.Context, p currency.Pair,
+	assetType asset.Item, timestampStart, timestampEnd time.Time) ([]trade.Data, error) {
+	req := AggregatedTradeRequestParams{
+		Symbol:    p,
+		StartTime: uint64(timestampStart.UnixMilli()),
+		EndTime:   uint64(timestampEnd.UnixMilli()),
+	}
+	trades, err := bi.GetAggregateTrades(ctx, &req)
+	if err != nil {
+		return nil, err
+	}
+	result := make([]trade.Data, len(trades))
+	exName := bi.Name
+	for i := range trades {
+		t := trades[i].toTradeData(p, exName, assetType)
+		result[i] = *t
+	}
+	return result, nil
+}
+
+// SubmitOrder submits a new order
+func (bi *Binanceus) SubmitOrder(ctx context.Context, s *order.Submit) (*order.SubmitResponse, error) {
+	var submitOrderResponse order.SubmitResponse
+	if err := s.Validate(); err != nil {
+		return nil, err
+	}
+	var timeInForce RequestParamsTimeForceType
+	if s.AssetType == asset.Spot {
+		var sideType string
+		if s.Side == order.Buy {
+			sideType = order.Buy.String()
+		} else {
+			sideType = order.Sell.String()
+		}
+		var requestParamOrderType RequestParamsOrderType
+		switch s.Type {
+		case order.Market:
+			requestParamOrderType = BinanceRequestParamsOrderMarket
+		case order.Limit:
+			timeInForce = BinanceRequestParamsTimeGTC
+			requestParamOrderType = BinanceRequestParamsOrderLimit
+		default:
+			return nil, errors.New(bi.Name + " unsupported order type")
+		}
+		var orderRequest = NewOrderRequest{
+			Symbol:           s.Pair,
+			Side:             sideType,
+			Price:            s.Price,
+			Quantity:         s.Amount,
+			TradeType:        requestParamOrderType,
+			TimeInForce:      timeInForce,
+			NewClientOrderID: s.ClientOrderID,
+		}
+		response, er := bi.NewOrder(ctx, &orderRequest)
+		if er != nil {
+			return nil, er
+		}
+		if response.OrderID > 0 {
+			submitOrderResponse.OrderID = strconv.FormatInt(response.OrderID, 10)
+		}
+		if response.ExecutedQty == response.OrigQty {
+			submitOrderResponse.Status = order.Filled
+		}
+		for i := range response.Fills {
+			submitOrderResponse.Trades = append(submitOrderResponse.Trades, order.TradeHistory{
+				Price:    response.Fills[i].Price,
+				Amount:   response.Fills[i].Qty,
+				Fee:      response.Fills[i].Commission,
+				FeeAsset: response.Fills[i].CommissionAsset,
+				Exchange: bi.Name,
+			})
+		}
+	} else {
+		return nil, fmt.Errorf("%s %w", s.AssetType, asset.ErrNotSupported)
+	}
+	return &submitOrderResponse, nil
+}
+
+// ModifyOrder will allow of changing orderbook placement and limit to
+// market conversion
+func (bi *Binanceus) ModifyOrder(ctx context.Context, action *order.Modify) (*order.ModifyResponse, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// CancelOrder cancels an order by its corresponding ID number
+func (bi *Binanceus) CancelOrder(ctx context.Context, order *order.Cancel) error {
```

Comment:
```suggestion
func (bi *Binanceus) CancelOrder(ctx context.Context, o *order.Cancel) error {
```
To not have a variable with the same name as a package name